File: tests/indexing/parsing/lang/fixtures/java/Comments.java
public class Comments {

    /**
     * Documentation
     */
    public int withComments(int a, int b) {
        // a leading line comment
        int sum = a + b; // a trailing line comment
        /* a block comment
           spanning lines */
        String url = "http://not-a-comment";
        return sum;
    }
}

File: tests/indexing/parsing/lang/fixtures/kotlin/Comments.kt
class Comments {

    /**
     * Documentation
     */
    fun withComments(a: Int, b: Int): Int {
        // a leading line comment
        val sum = a + b // a trailing line comment
        /* a block comment
           spanning lines */
        val url = "http://not-a-comment"
        return sum
    }
}
